### ISSUE TITLE
Regarding Audio Length for Train  and Test

### ISSUE THREAD

**Issue by 009deep** [miquelindia90/DoubleAttentionSpeakerVerification#5]
I trained my model using both _MHA_  and _DoubleMHA_, and what I observed is if I use different lengths of audio samples in training and testing then test evaluation results are really bad, close to random. For example, if my training audio length is 2s to get sensible results I need to use test audio of 2s as well. I can see in the model that no. of trainable parameters don't rely on audio length then why such a difference in evaluation?  Is my observation correct? Are you facing a similar problem?

**Comment by miquelindia90** [comment]
Hi. I've never faced this problem. In fact the pre-trained model was trained using 350 frame segments and evaluated with larger audios. However i never trained these models with 200 frames. Could you tell me how long are the audios you are using for test ? Have you tried to use the pre-trained model for the test. Does it have this same bad behaviour ?

**Comment by 009deep** [comment]
I haven't used the pre-trained model. One of the test sets I  am using is voxceleb test set which has a varied length of audio with an average length of 7s. Another internal test set has an average length of below 3s. Each audio has a different length in test samples. I'll take a second look at my implementation to make sure I've integrated implementation correctly. Thanks.

**Comment by miquelindia90** [comment]
The pre-trained model should work well in VoxCeleb 1 test sets. If is not working I will check the implementation.
